projects/laji-api-client-b/src/laji-api-client-b.service.ts

```diff
@@ -171,5 +249,18 @@ export class LajiApiClientBService {
     }
     return allLevels;
   }
+
+  private paramsWithDefaults<P extends Path, M extends Method>(
+    params: ParametersWithOptionalQueryKeys<Parameters<paths[P][M]>, 'personToken' | 'lang'>
+  ): Parameters<paths[P][M]> {
+    return {
+      ...(params || {}),
+      query: {
+        lang: this.lang,
+        personToken: this.personToken,
```

Comment:
There are endpoints in trait-db with optional personToken parameters :/ so the automatic population would break intentionally not wanting to supply personToken

Reply:
Hmm, that's problematic indeed. With this PR you can prevent the `personToken` being populated by adding `personToken: undefined` to the query, but it could create confusion.

But I'd like to move the `personToken` to headers anyway in the future. Whether for example `/trait/dataset-permissions` returns the user's permissions or the systems permissions could be controlled with a separate query param... I'll talk with Esko how to handle this.